Resources/Prototypes/Loadouts/mask.yml

```diff
@@ -0,0 +1,113 @@
+- type: loadout
+  id: LoadoutMaskSterile
+  category: Mask
+  cost: 1
+  exclusive: true
+  items:
+    - ClothingMaskSterile
+
+- type: loadout
+  id: LoadoutMaskMuzzle
+  category: Mask
+  cost: 2
+  exclusive: true
+  items:
+    - ClothingMaskMuzzle
+
+- type: loadout
+  id: LoadoutMaskGas
+  category: Mask
+  cost: 3
```

Comment:
I don't know................ people can just find this in maints easily instead of sinking 3 points.

Reply:
Some people would like to start with a gas mask instead of immediately maints diving at the start of the shift, so I'd like to give them that choice.